pkg/nats/transport/nats.go

```diff
@@ -70,6 +83,28 @@ func (c *NATSTransportConfig) Validate() error {
 	} else {
 		mErr = errors.Join(mErr, validate.IsNotEmpty(c.Orchestrators, "missing orchestrators"))
 	}
+
+	serverCertProvided := len(c.ServerTLSCert) > 0
+	serverKeyProvided := len(c.ServerTLSKey) > 0
```

Comment:
not sure if this is the preferred way to check for blank strings in go, but should this be `c.ServerTLSCert != ""`

Reply:
I can go with `c.ServerTLSCert != ""` if it is more conventional.

Reply:
Fixed